### ISSUE TITLE
scor.generator as new repository please

### ISSUE THREAD

**Issue by seebi** [vocol/scor#20]
Ich halte das für keine gute Idee, alles in ein Repo zu packen. besser ein scor.generator project mit extra README.md usw.

**Comment by np00** [comment]
ja, so richtig gefällt es mir auch nicht. Nach der review-deadline werde ich es verschieben.
